lib/integration/crud.ts

```diff
@@ -244,6 +246,28 @@ async function _getFormByStatus(
   return [];
 }
 
+const _getForms = async (templates: unknown[] = [], limit = 50, offset = 0): Promise<unknown[]> => {
+  try {
+    const lambdaResult = await crudTemplates({ method: "GET", limit: limit, offset: offset });
+
+    if (!lambdaResult?.data?.records) {
+      return templates;
+    }
+
+    Array.prototype.push.apply(templates, lambdaResult.data.records);
```

Comment:
Since you resolved the convo for visibility I will re-comment here. I'm not sure exactly what you mean by "concat doesn't do a deep copy`. Concat returns the amalgamation of two arrays which is completely new. If you're talking about the individual items I don't think applying the push function on each of the items would result in a deep copy of the individual items.

Reply:
This would be much simpler to read 

`templates = templates.concat(lambdaResult.data.records);`

Reply:
Ack! I mis-remembered that `.concact` does not affect the original array, in the way that `.push` does. 

From the docs:`This method does not change the existing arrays, but instead returns a new array.`